src/client/components/OfferProductsList/OfferProductsList.component.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Product from '../SpecialOffers/OfferProductModel.component';
+import './OfferProductsList.styles.css';
+
+export default function OfferProducts({ products }) {
+  const addToCart = () => {
```

Comment:
`addToCart` and `addFavorites` should be a prop to this presentational component.